ios/Classes/Blemulator/Bridging/DartMethodCaller.m

```diff
@@ -18,4 +21,26 @@ - (instancetype)initWithDartMethodChannel:(FlutterMethodChannel *)dartMethodChan
     return self;
 }
 
+// MARK: - Methods
+
+- (void)createClient {
+    [self.dartMethodChannel invokeMethod:DART_METHOD_NAME_CREATE_CLIENT
+                               arguments:nil
+                                  result:[self simpleInvokeMethodResultHandlerForMethod:DART_METHOD_NAME_CREATE_CLIENT]];
+}
+
+// MARK: - Utility methods
+
+- (InvokeMethodResultHandler)simpleInvokeMethodResultHandlerForMethod:(NSString *)methodName {
+    return ^(id _Nullable result) {
+        if ([result class] == [FlutterError class]) {
+            NSLog(@"%@%@%@%@", @"DartMethodCaller.", methodName, @": FlutterError: ", [(FlutterError *)result message]);
```

Comment:
What's the curly hair? `%@%@%@%@`

Reply:
It's for specyfing the format of the arguments in output of NSLog.